TFNetworkRecLayer.py

```diff
@@ -2660,6 +2660,7 @@ def get_state_by_key(cls, state, key, shape=None):
     """
     :param tf.Tensor|tuple[tf.Tensor]|namedtuple state:
     :param int|str|None key:
+    :param tuple shape: Shape of the state.
```

Comment:
You should specify also what the type is, e.g. `tuple[int]`.

Reply:
Or `tuple[int|None]`. Or `tuple[int|tf.Tensor]`. Not sure which one it is here.

Reply:
Done. Specified as `tuple[int|None]`